README.md

```diff
@@ -0,0 +1,6 @@
+# [Stackdriver Monitoring Agent](https://cloud.google.com/monitoring/agent/)
+
+This is Stackdriver Monitoring agent's implementation. It is forked from collectd ([official website](https://collectd.org/), [repo](https://github.com/collectd/collectd)).
```

Comment:
Let's do `This is the implementation of the Stackdriver monitoring agent, forked from [collectd](http://collectd.org).`
The "official website" is noise, and the repo pointer is actually at the top of this one, so no point in repeating it in the README.